services/workflows-service/src/business/business.service.ts

```diff
@@ -46,6 +46,13 @@ export class BusinessService {
     return await this.repository.findById(id, args, projectIds);
   }
 
+  async getByCorrelationIdUnscoped(
```

Comment:
Why unscoped?

Reply:
The alternative would be adding `projectId` as a search query param. To my understanding this is how we do it in other places.